### ISSUE TITLE
Idempotency

### ISSUE THREAD

**Issue by shelbyd** [uesteibar/reviewer-lottery#36]
Currently, each time this is run it will potentially add new reviewers. I'd like to run this after all our tests pass.

It would be great if this could check the current set of reviewers on the PR and only assign to fulfill the constraints. That way, running it subsequently would result in no new assignments.

This seems like a pretty substantial change, so would probably be better to have someone more familiar with the project do it.

**Comment by uesteibar** [comment]
I agree it'd be useful yeah. Tho I doubt I'll have time to do it myself in the near future :(
